SCAutolib/models/file.py

```diff
@@ -311,33 +311,30 @@ class SoftHSM2Conf(File):
     _content = None
     _card_dir = None
 
-    def __init__(self, filepath: str, card_dir: str):
+    def __init__(self, filepath: Union[str, Path], card_dir: Union[str, Path]):
```

Comment:
Changes in file.py SoftHSM2Conf class are not strictly related to implementation of Cards. Most of them are improvements of older code. I would prefer to have these in separate commit. If we for whatever reason revert commit "[Add functionality for virtual smart card](https://github.com/x00Pavel/SCAutolib/pull/65/commits/ba7c492c5dd4a67b4920b359813ccb4fdfb7d567)" we would lost also these improvements. That would be pity.